File: decompiled/sources/com/hjq/http/ssl/UnSafeTrustManager.java
package com.hjq.http.ssl;

import java.security.cert.X509Certificate;
import javax.net.ssl.X509TrustManager;

public final class UnSafeTrustManager implements X509TrustManager {
    public void checkClientTrusted(X509Certificate[] x509CertificateArr, String str) {
    }

    public void checkServerTrusted(X509Certificate[] x509CertificateArr, String str) {
    }

    public X509Certificate[] getAcceptedIssuers() {
        return new X509Certificate[0];
    }
}

File: decompiled/sources/com/hjq/http/ssl/UnSafeTrustManager.kt
package com.hjq.http.ssl

import java.security.cert.X509Certificate
import javax.net.ssl.X509TrustManager

class UnSafeTrustManager : X509TrustManager {
    override fun checkClientTrusted(x509CertificateArr: Array<out X509Certificate>?, str: String?) {
    }

    override fun checkServerTrusted(x509CertificateArr: Array<out X509Certificate>?, str: String?) {
    }

    override fun getAcceptedIssuers(): Array<X509Certificate> {
        return emptyArray()
    }
}
